lib/ecto/adapters/myxql/connection.ex

```diff
@@ -572,6 +572,11 @@ if Code.ensure_loaded?(MyXQL) do
       quote_name(literal)
     end
 
+    defp expr({:alias, _, [column_expr, name]}, sources, query) do
```

Comment:
I think we should explicitly check for “alias/2” at the root of each entry of query.select.fields. If alias is anywhere else, we should raise.